### ISSUE TITLE
Reorder Page Elements for Asset Page

### ISSUE THREAD

**Issue by hanson-lam** [VizerISD/personalhealthcarewallet#63]
While it's nice to see who has accessed your information, it's probably more important to actually **see the information**. As such, many users have suggested that we change the order on the asset page. We should try to move the box containing the "download" and "asset access" tabs to the top of the page. Additionally, reordering so that the description and metadata box is above the graphics might be nice as well.

TLDR. We should move some of the bottom elements of the asset page to the top. The graphs should be at the bottom.

<img width="834" alt="Screenshot 2023-04-18 at 6 06 37 PM" src="https://user-images.githubusercontent.com/71535598/232915685-f75adfdd-93ee-4fdc-8d4e-83fe0efcef2e.png">

**Comment by IanMacDougall** [comment]
@CarolineRinks this might be a good task for you to take since you have pretty good experience working with CSS and the asset page. I discussed this with @hanson-lam, but one idea for the new layout could be...

TITLE
______________________________
|  ASSET ACCESS  | DOWNLOAD   |
| _______________|_____________|
______________________________
|               DESCRIPTION                  | 
|_____________________________|

                        VIZ 1
                        VIZ 2

I think the main thing would be putting the asset access part at the top or as one of the first things the user sees on the asset page. The description box, viz 1, and viz 2 can be interchanged based on what you think. If anyone else has a different suggestion add it here!

**Comment by CarolineRinks** [comment]
Yeah, I can work on this one! @IanMacDougall
